MULTIPLATFORM.md

```diff
@@ -0,0 +1,244 @@
+# Kotlin/Native in multiplatform projects
+
+*TODO: Some introduction*
+
+## Creating multiplatform Android/iOS application with Kotlin
+
+*TODO: High-level application structure: what code is common (data, logic) and what code is platform (io, ui, network).
+What languages are used in what parts of the project.*
+
+*TODO: link to the calculator project*
+
+*TODO: link to the multiplatform description*
+
+### 1. Preparing a workspace
+
+Our multiplatform application will include three parts:
+
+ * An **Android application** represented by a separate Android Studio project.
+ * An **iOS application** represented by a separate XCode project.
+ * A **multiplatform library** represented by a separate Gradle-build and containing a business logic of the application.
+   This library can contain both platform-dependent and platform-independent code and is compiled into a `jar`-library
+   for Android and in a framework for iOS.
+
+In its turn, the multiplatform library will include three subprojects:
+
+ * `common` - contains a common logic for both applications;
+ * `ios` - contains an iOS-specific code;
+ * `android` - contains an Android-specific code.
+
+Represent this structure as a directory tree. Assume that our multiplatform library is intended to generate different
+greetings on different platform. Create the following directory structure:
+
+    application/
+    ├── androidApp/
+    ├── iosApp/
+    └── greeting/
+        ├── common/
+        ├── android/
+        └── ios/
+
+Now we have a basic structure of the multiplatform application and can proceed to implementing of the multiplatform library.
+
+### 2. Multiplatform library
+
+*Note: It's highly recommended to use [Gradle wrapper](https://docs.gradle.org/current/userguide/gradle_wrapper.html)
+to run the build. To create the wrapper, just execute `gradle wrapper` in the `greeting` directory. After that you can
```

Comment:
You talk about the gradle wrapper here but hasn't explain that we use gradle "really".
E.g. how to install gradle etc.

Reply:
Point is that with Gradle wrapper there's no need to install Gradle :).

Reply:
But this **note** says that you can execute `gradle wrapper` which means you have/should install gradle first.
Don't know if this will be just a documentation or later a real project. 
If it's just a documentation than I would make a "special" section about gradle like

> ### Gradle
> We are using Gradle as our build system. It is highly recommended to use the Gradle wrapper for your builds. More information about it can be found here ... (link to the docu)

Reply:
Agree. I'll add such a section in the doc.

Reply:
Done.